ui/src/pages/publications/[id]/flag/[flagId]/index.tsx

```diff
@@ -82,12 +80,11 @@ type Props = {
 
 const FlagThread: Next.NextPage<Props> = (props): JSX.Element => {
     const router = Router.useRouter();
-    const { data, isValidating, error, mutate } = useSWR<Axios.AxiosResponse<Interfaces.FlagWithComments>>(
-        props.flagUrl,
-        {
-            fallback: props.fallback
-        }
-    );
+    const { data, isValidating, error, mutate } = useSWR<Interfaces.FlagWithComments>(props.flagUrl, {
+        fallback: props.fallback
+    });
+
+    console.log(data);
```

Comment:
This console log can probably be removed

Reply:
yep, removed.